modules/appeals_api/app/swagger/appeals_api/v1/notice_of_disagreements_controller_swagger.rb

```diff
@@ -8,6 +8,7 @@ class AppealsApi::V1::NoticeOfDisagreementsControllerSwagger
   PATH_ENABLED_FOR_ENV = Settings.modules_appeals_api.documentation.notice_of_disagreements_v1
 
   NOD_TAG = ['Notice of Disagreements'].freeze
+  OBJ = :object
```

Comment:
Actually I guess the better question is: why was this constant added? I'm not seeing it used in the file or any other part of the changes. Is there something I'm missing about the changes that requires this constant be present?

Reply:
Whoops, I pulled this in originally because it was in the HLR one and didn't double check if I actually needed it. Good catch!